connectors/airwatch/src/test/java/com/vmware/connectors/airwatch/AirWatchControllerTest.java

```diff
@@ -323,6 +323,7 @@ private void testRequestCards(String requestFile, String responseFile, String ac
                 .collect(Collectors.joining())
                 .map(JsonNormalizer::forCards)
                 .block();
+        body = body.replaceAll("[a-z0-9]{40,}", "test-hash");
```

Comment:
I changed all of the connector tests to ignore their hash.  I don't want a test failing in almost every connector every time we add a significant property to cards.